helpers/pyband/pyband/data.py

```diff
@@ -24,7 +25,7 @@ def as_json(self) -> dict:
 
     def validate(self) -> bool:
         if self.amount < 0:
-            raise ValueError("Expect amount more than 0")
+            raise InsufficientCoinError("Expect amount more than 0")
```

Comment:
I think it can use ValueError or NegativeIntegerError (a coin cannot be less than zero)